envs/browser/transport.js

```diff
@@ -17,22 +17,31 @@ class Transport extends Obfuscated {
       this.dc.test ? '/apiws_test' : '/apiws'
     }`;
     this.crypto = crypto;
+    this.destroyed = false
+    this.handleError = this.handleError.bind(this)
+    this.handleOpen = this.handleOpen.bind(this)
+    this.handleClose = this.handleClose.bind(this)
+    this.handleMessage = this.handleMessage.bind(this)
 
     this.connect();
   }
 
+  destroy() {
+    this.destroyed = true
+    this.ws.close()
```

Comment:
Please stick to the style. Add a commas and semicolons

Reply:
fixed here https://github.com/alik0211/mtproto-core/pull/235/commits/185d52e3ffbdc65bcfe7b69d810a0b4a4ff87446